tools/donate-cpu.py

```diff
@@ -221,15 +222,15 @@ def scanPackage(workPath, cppcheck, jobs):
     print('Analyze..')
     os.chdir(workPath)
     libraries = ' --library=posix'
-    if hasInclude('temp', '<wx/') or hasInclude('temp', '"wx/'):
+    if hasInclude('temp', ['<wx/', '"wx/']):
```

Comment:
Not exactly addressing this PR, but: using `<>` vs. `""` should be supported the same way for all headers.
If we want to "promote" using `<>` then wxWidgets should be no exception.

Reply:
The intention is to limit false detection of library includes. I don't really like that if some user header is put in a folder "wx" in a project and you include that then we will assume wxwidgets is used. I guess this pattern needs to be more strict.

Reply:
> If we want to "promote" using <> then wxWidgets should be no exception.

yes. hmm I added this even though I really don't like it. I did it because this misuse is apparently common practice.